src/java.desktop/share/classes/javax/swing/plaf/basic/BasicSplitPaneDivider.java

```diff
@@ -299,7 +299,14 @@ public boolean isMouseOver() {
     }
 
     /**
-     * Returns dividerSize x dividerSize
+     * Returns the preferrred {@code Dimension} of the divider.
+     * If the splitpane is HORIZONTAL_SPLIT, the preferred size is obtained from
+     * width of {@code getDividerSize} pixels and height of 1 pixel
+     * If the splitpane is VERTICAL_SPLIT, the preferred size is obtained from
+     * height of {@code getDividerSize} pixels and width of 1 pixel
+     *
+     * @return a {@code Dimension} object containing the preferred size of
+     *         {@code BasicSplitPaneDivider}
```

Comment:
I wonder how we usually specify the getPref/getMin methods. Do we really describe how it is implemented or just mention something like "Returns the minimum size for the XXX".

Reply:
There's instances of both so I took the liberty to err on the more elaborate way so as to not leave anything ambiguous.

Reply:
But then it will not be possible to change it in the future here or in any subclasses to say 2 pixels, etc.

Reply:
We can then add @implSpec if you wish